srcpkgs/yabridge/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'yabridge'
+pkgname=yabridge
+version=4.0.2
+revision=1
+archs="x86_64"
+build_style=meson
+hostmakedepends="ninja pkg-config cmake grep git $(vopt_if with_32bit "gcc-multilib")"
```

Comment:
https://github.com/void-linux/void-packages/pull/39575#discussion_r985083246

Reply:
grep is required as the chroot-grep causes the following error:
ompiler stderr:

Running command: /usr/bin/sh -c wine --version | grep --only-matching -P '[0-9]+\.[0-9]+' | head -n1
--- stdout ---

--- stderr ---
grep: Perl matching not supported in a --disable-perl-regexp build

grep is version 3.8 has has perl-regexp enabled so the build works.  
I moved grep from hostmakedepends to make depends.

Reply:
hostmakedepends is correct here, but we can't have grep there currently anyway

Reply:
ok.. changed it back to host depends.

Reply:
it can not be there either

Reply:
so if it cant be in hostmakedepends or makedepends ?????

Reply:
It can not be in build dependencies because it would break the masterdir. This wasn't an issue in any package so far.

Reply:
You can use a patch to change the `-P` in https://github.com/robbert-vdh/yabridge/blob/5b29ac386e346666b1407c31639c53ec99bf63e6/meson.build#L154 into `-E` since `-P` is strictly speaking not needed anymore (it's an artifact from a pervious version that required lookaround). I've never encountered a `grep` on Linux without PCRE support before though.

Reply:
Ive done a patch using sed.   I can change it to grep -E is more keeping with your origin code?